package.json

```diff
@@ -33,9 +33,8 @@
   },
   "dependencies": {
     "ds-test": "github:dapphub/ds-test#e282159",
```

Comment:
should this be remove as well?

Reply:
we cannot, because forge-std/Test.sol depends on this one